### PR TITLE
setting up  project directory structure & database

### PR DESCRIPTION
**Pull request by gik-hub**
#### What does this PR do?
This PR is meant to set up the project with babel, eslint, .gitignore, and a pull request template 
set up the users model database structure migrations
#### Description of Task to be completed?
1. Set up a babelrc file to convert the es6 file to es5
2. set up an eslintrc file to ensure the code is in es6.
3. Add a .gitignore file
4. add a PR template to ease the process of a PR creation
5. Add users model migration 
#### How should this be manually tested?
1) code review
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168022333
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by mucyomiller — COMMENTED** [review]
- @didasmbalanya  can you help us make develop branch the default branch because from what I have seen we tend to forget and create a pull request with the master branch as base branch likes this one @TgeeT forgot to select develop branch as base branch I had to change it thanks! 
- Let's not forget to also stick to Andela  [Git-naming-conventions-and-best-practices](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices)

**Comment by gik-hub** [comment]
> * @didasmbalanya  can you help us make develop branch the default branch because from what I have seen we tend to forget and create a pull request with the master branch as base branch likes this one @TgeeT forgot to select develop branch as base branch I had to change it thanks!
> * Let's not forget to also stick to Andela  [Git-naming-conventions-and-best-practices](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices)

hey, @mucyomiller, there is a develop branch which I did a checkout from. while creating the ch-project-setup branch

File: src/config/config.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+require("@babel/register");
+
+require('dotenv').config();
+
+console.log(process.env.DB_USER)
```
Comment (Jordybastien) [inline src/config/config.js:5]: Remove logs

File: package.json [hunk 2]
```diff
@@ -15,5 +15,21 @@
   "bugs": {
     "url": "https://github.com/didasmbalanya/E-petition/issues"
   },
-  "homepage": "https://github.com/didasmbalanya/E-petition#readme"
```
Comment (Jordybastien) [inline package.json:18]: there is a repetition here

**Pull request merged**